### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Brenda309**
### Project requirement
Create a new app. Give it a meaningful name.
Initialize your project with Git.
Make sure that your project has Postgres database set up.
Build your project schema.
Create and run the necessary migration files.
Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
Foreign keys should be included.
All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Brenda, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect gitflow. Kindly note that the gitflow of this project should have merge request **from feature into development**. Kindly use the correct gitflow, and request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit.

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    t.index ["post_id"], name: "index_likes_on_post_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "title"
+    t.string "text"
+    t.integer "commentsCounter"
+    t.integer "likesCounter"
```
Comment (TedLivist) [inline db/schema.rb:42]: - [ ] **[OPTIONAL]** In keeping with convention I would recommend you rename these attributes in the posts table to `comments_counter` and `likes_counter` respectively.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Brenda309,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 1 above)
Comment (Brenda309) [reply db/schema.rb:42]: Thank you for the suggestions. I will work on that.

**Pull request merged**